app/models/ability.rb

```diff
@@ -10,18 +8,50 @@ def initialize(user)
 
     alias_action :create, :read, :update, :destroy, to: :crud
 
+    # unconfirmed user
+    can :read, User
+    can :update, User, id: user.id
+    can :resend_confirmation_instruction, User, id: user.id
+    can :read_email, User, hide_email: false # view helper
+    can :read, Team
+    can :read, Project
+    can :read, :feed_entry
```

Comment:
First off: forgive my questions, I'm not familiar with CanCan - I'm always using Pundit... 

What does this syntax here mean? `:feed_entry` is a symbol, not a class / model... ❓

Reply:
I <3 your questions. 
It is an :kind of an Activity, and used in lib/feed/item. I left it in here for now, because I am not changing abilities until they are thoroughly tested 😇